Formula/m/markdownlint-cli.rb

```diff
@@ -36,7 +36,7 @@ def install
 
       body
     EOS
-    assert_match "MD022/blanks-around-headings/blanks-around-headers",
+    assert_match "MD022/blanks-around-headings Headings should be surrounded by blank lines",
```

Comment:
- relates to https://github.com/DavidAnson/markdownlint/commit/a9a77940c5502e21ed0b2a3ac5976dbcc0d5c52a